src/components/ArticleVotes.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from 'react';
+
+import { patchArticleVotes } from '../utils';
+
+export default function ArticleVotes({article, setArticle}) {
+  const [voteChange, setVoteChange] = useState(0);
+
+  function handleVote(incVote) {
+    setArticle(article => ({ ...article, votes: article.votes + incVote }));
+    patchArticleVotes(article.article_id, incVote);
+    setVoteChange(currVoteChange => currVoteChange + incVote);
```

Comment:
In case something goes wrong with the request (for example when the app is offline) please make sure that the displayed votes get reverted and the user get's a message that something went wrong.